docs/Routes.md

```diff
@@ -32,6 +32,7 @@ They need to be in
 * `logLevel`: set log level for this route. See below.
 * `config`: object used to store custom configuration.
 * `version`: a [semver](http://semver.org/) compatible string that defined the version of the endpoint. [Example](https://github.com/fastify/fastify/blob/versioned-routes/docs/Routes.md#version).
+* `prefixRootOnly`: when using a `/` url with a prefix, setting this option to true will register only `/prefix` without trailing slash. (Defaults to `false`)
```

Comment:
Not sure the option name is clear enough.
How about `disablePrefixTrailingSlash`? (Not sure about this one either)

Reply:
I'm open to anything, to be honest naming this option was the hardest part of the PR 😄

Reply:
Agree! I also think we should add a code example to explain the behavior of the option.
Which usually is better than thousand of words :D

Reply:
I think we really would like three options here:

* define both `/path`  and `/path/` (default)
* define `/path`
* define `/path/`

The default is picked because `http://localhost:3000/`  and `http://localhost:3000` are going to the same route _anyway_.

Reply:
I was thinking about this, perhaps a URL of `/` should only refer to `{prefix}/` and another token for url such as `.` representing just the prefix?

Reply:
Unfortunately `http://localhost:3000/.` is a valid URL. We cannot use `.`.